koji/vars/users.yml

```diff
@@ -78,48 +39,46 @@ enabled_users:
   - name: kojira.katello.org
     permissions:
       - build
+      - repo
   - name: lzapleta
     permissions:
       - admin
```

Comment:
I recall @lzap wanted to hand over Koji maintenance. Do you want to keep admin access or is it sufficient to have regular build/repo permissions?